main.js

```diff
@@ -0,0 +1,146 @@
+console.log("hi from js")
+
+// Input Box
+$inputBox = document.querySelector('#input-box');
+// Start Button
+$startButton = document.querySelector('#start-button');
+
+// Time-Keeper area
+$timeKeeper = document.querySelector('#time-keeper');
+
+var timerMinutes = 0;
+var timerSeconds = 10;
+
+var displayTaskandStartTimer = function () {
+  if ($inputBox.value === "") {
+    return false;
+  }
+  createTaskandTimer();
+  // Start Timer
+  $timerSecondsSpan = document.getElementById('timerSeconds');
+  $timerMinutesSpan = document.getElementById('timerMinutes');
+  clockStart();
+
+};
+
+// Create Event Listener on Button to create prompt of timer and task
+$startButton.addEventListener('click', displayTaskandStartTimer);
+
+
+// Start 25 Minute timer //
+var clockStart = function () {
+  myAudio = new Audio ('Tetrishead.mp3');
+  myAudio.play();
+  clockId = setInterval(countDown,1000);
+}
+
+
+//timer Second logic
+var countDown = function() {
+  console.log('Time: ' + timerMinutes + ' Minutes ' + timerSeconds + ' Seconds');
+  if (timerMinutes !== 0 && timerSeconds == 0) {
+    timerSeconds = 59;
```

Comment:
Looks like you're formatting seconds at 59. That works! You could also just store the time in milliseconds 25 * 60 * 1000 and then format that number as a string.